lesson13/main.js

```diff
@@ -0,0 +1,106 @@
+const openBtn = document.getElementById("js-open-modal-btn");
+const modal = document.getElementById("js-modal");
+const closeBtn = document.getElementById("js-close-btn");
+const overlay = document.getElementById("js-overlay");
+const ul = document.getElementById("js-ul");
+const openListBtn = document.getElementById("js-open-list-button")
+
+
+const renderLoading = () => {
+  const loading = document.getElementById("js-loading");
+  const loadingImg = document.createElement("img");
+  loadingImg.src = "img/loading.gif";
+  loadingImg.id = "loading-img";
+  loading.appendChild(loadingImg);
+};
+
+const removeLoading = () => {
+  document.getElementById("loading-img").remove();
+};
```

Comment:
nits
It can be written like this
```suggestion
const removeLoading = () => document.getElementById("loading-img").remove();
```

Reply:
Thank you for the suggestions Haru-san,
I think I got a similar suggestion from you before and it totally slipped out of my memory.
Please check my latest commit 293e2580a0e6ee145e434b400f3e26510fc1ba4d